hadoop-ozone/dist/src/main/k8s/ozone/om-statefulset.yaml

```diff
@@ -44,8 +44,8 @@ spec:
             - om
             - --init
           env:
-            - name: "WAITFOR"
-              value: "scm-0.scm:9876"
+            - name: "ENSURE_OM_INITIALIZED"
```

Comment:
ENSURE_OM_INITIALIZED is not required here. For kubernetes we follow a different (more generic) approach. We can define an 'initContainer' to execute the 'om --init' first. (as you can see this is the initContainer here). As the 'om --init' can be executed multiple times it works well without the environment variable (which is handled by the starter script of the container.

Reply:
Removed in the latest commit.